docs/source/example_notebooks/dowhy_generating_synthetic_datasets.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
you may also consider removing this notebook, now that the information is in the docstring.